content/applications/sales/sales/send_quotations/quote_template.rst

```diff
@@ -1,93 +1,121 @@
-=======================
-Use quotation templates
-=======================
+===================
+Quotation templates
+===================
 
-By creating custom quotation templates, you will save a lot of time. Indeed, with the use of
-templates, you will be able to send complete quotations at a fast pace.
+Creating custom quotation templates saves a lot of time. By using preconfigured quotation
+templates, completed quotations are able to be sent at a much faster pace.
 
 Configuration
 =============
 
-To enable this feature, go to :menuselection:`Sales --> Configuration --> Settings` and activate
-**Quotation Templates**.
+In order to create custom quotation templates, the *Quotation Template* feature must be enabled. To
+do that, navigate to :menuselection:`Sales app --> Configuration --> Settings`, and scroll to the
+:guilabel:`Quotations & Orders` heading.
 
-.. image:: quote_template/quotations_templates_1.png
+In that section, check the box beside the :guilabel:`Quotation Templates` feature. Doing so reveals
+a new :guilabel:`Default Template` field, in which a default quotation template can be chosen from a
+drop-down menu.
+
+.. image:: quote_template/quotations-templates-setting.png
    :align: center
-   :class: img-thumbnail
-   :alt: How to enable quotation templates on Odoo Sales?
+   :alt: How to enable quotation templates on Odoo Sales.
+
+Also, upon activating the :guilabel:`Quotation Template` feature, a direct link to the
+:guilabel:`Quotation Templates` page appears beneath the :guilabel:`Default Template` field of the
+settings page. Clicking that link reveals the :guilabel:`Quotation Templates` page, from which
```

Comment:
The text "Settings" is displayed as the page title and it is within the active instruction– so a guilabel is needed :)
```suggestion
:guilabel:`Settings` page. Clicking that link reveals the :guilabel:`Quotation Templates` page, from
which templates can be created, viewed, and edited.
```